SPECS/cmake/macros.cmake

```diff
@@ -26,7 +26,8 @@
 %if "%{?_lib}" == "lib64" \
 	%{?_cmake_lib_suffix64} \\\
 %endif \
-	-DBUILD_SHARED_LIBS:BOOL=ON
+	-DBUILD_SHARED_LIBS:BOOL=ON \\\
```

Comment:
I think we should call this out in the description of the PR and title of the PR that we are changing a `cmake` macro.

Although this change seems harmless, but still we should call it out.